<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window</class>
 <widget class="QMainWindow" name="Window">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>640</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>640</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>640</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Bahnschrift SemiBold</family>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:#091833</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="Calc">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>40</number>
    </property>
    <property name="topMargin">
     <number>20</number>
    </property>
    <property name="rightMargin">
     <number>40</number>
    </property>
    <property name="bottomMargin">
     <number>20</number>
    </property>
    <item row="5" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QPushButton" name="pBtn_1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>90</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Bahnschrift SemiBold</family>
        <pointsize>25</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background:#133e7c; color:#0abdc6; border: 1px solid #0abdc6</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QPushButton" name="pBtn_cls">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>90</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Bahnschrift SemiBold</family>
        <pointsize>25</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background:#711c91; color:#d600ff; border: 1px solid #d600ff</string>
      </property>
      <property name="text">
       <string>C</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QPushButton" name="pBtn_opadd">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>90</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Bahnschrift SemiBold</family>
        <pointsize>25</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background:#711c91; color:#d600ff; border: 1px solid #d600ff</string>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QPushButton" name="pBtn_lpar">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>90</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Bahnschrift SemiBold</family>
        <pointsize>25</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background:#711c91; color:#d600ff; border: 1px solid #d600ff</string>
      </property>
      <property name="text">
       <string>(</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QPushButton" name="pBtn_5">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>90</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Bahnschrift SemiBold</family>
        <pointsize>25</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background:#133e7c; color:#0abdc6; border: 1px solid #0abdc6</string>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QPushButton" name="pBtn_opred">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>90</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Bahnschrift SemiBold</family>
        <pointsize>25</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background:#711c91; color:#d600ff; border: 1px solid #d600ff</string>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QPushButton" name="pBtn_opdiv">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>90</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Bahnschrift SemiBold</family>
        <pointsize>25</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background:#711c91; color:#d600ff; border: 1px solid #d600ff</string>
      </property>
      <property name="text">
       <string>รท</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QPushButton" name="pBtn_del">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>90</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Bahnschrift SemiBold</family>
        <pointsize>25</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background:#711c91; color:#d600ff; border: 1px solid #d600ff</string>
      </property>
      <property name="text">
       <string>&lt;&lt;</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QPushButton" name="pBtn_8">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>90</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Bahnschrift SemiBold</family>
        <pointsize>25</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background:#133e7c; color:#0abdc6; border: 1px solid #0abdc6</string>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QPushButton" name="pBtn_0">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>90</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Bahnschrift SemiBold</family>
        <pointsize>25</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background:#133e7c; color:#0abdc6; border: 1px solid #0abdc6</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QPushButton" name="pBtn_dot">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>90</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Bahnschrift SemiBold</family>
        <pointsize>25</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background:#711c91; color:#d600ff; border: 1px solid #d600ff</string>
      </property>
      <property name="text">
       <string>.</string>
      </property>
     </widget>
    </item>
    <item row="6" column="3" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QPushButton" name="pBtn_eq">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>90</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Bahnschrift SemiBold</family>
        <pointsize>25</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background:#4d1b7b; color:#c79dd7; border: 1px solid #c79dd7</string>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QPushButton" name="pBtn_3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>90</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Bahnschrift SemiBold</family>
        <pointsize>25</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background:#133e7c; color:#0abdc6; border: 1px solid #0abdc6</string>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QPushButton" name="pBtn_4">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>90</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Bahnschrift SemiBold</family>
        <pointsize>25</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background:#133e7c; color:#0abdc6; border: 1px solid #0abdc6</string>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QPushButton" name="pBtn_9">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>90</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Bahnschrift SemiBold</family>
        <pointsize>25</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background:#133e7c; color:#0abdc6; border: 1px solid #0abdc6</string>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QPushButton" name="pBtn_7">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>90</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Bahnschrift SemiBold</family>
        <pointsize>25</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background:#133e7c; color:#0abdc6; border: 1px solid #0abdc6</string>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QPushButton" name="pBtn_6">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>90</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Bahnschrift SemiBold</family>
        <pointsize>25</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background:#133e7c; color:#0abdc6; border: 1px solid #0abdc6</string>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QPushButton" name="pBtn_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>90</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Bahnschrift SemiBold</family>
        <pointsize>25</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background:#133e7c; color:#0abdc6; border: 1px solid #0abdc6</string>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QPushButton" name="pBtn_rpar">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>90</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Bahnschrift SemiBold</family>
        <pointsize>25</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background:#711c91; color:#d600ff; border: 1px solid #d600ff</string>
      </property>
      <property name="text">
       <string>)</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QPushButton" name="pBtn_opmul">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>90</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Bahnschrift SemiBold</family>
        <pointsize>25</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background:#711c91; color:#d600ff; border: 1px solid #d600ff</string>
      </property>
      <property name="text">
       <string>ร</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="4" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QLabel" name="screen">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>100</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Bahnschrift SemiBold</family>
        <pointsize>40</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#0abdc6</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
